libs/header/p2p/helpers.go

```diff
@@ -20,7 +20,7 @@ func protocolID(protocolSuffix string) protocol.ID {
 	return protocol.ID(fmt.Sprintf("/header-ex/v0.0.3/%s", protocolSuffix))
 }
 
-func pubsubTopicID(protocolSuffix string) string {
+func PubsubTopicID(protocolSuffix string) string {
```

Comment:
Why exported though?

Reply:
It will be required for the setup

As you can see here, we don't set/use scores anywhere in this PR. Using setting those will be done in another PR.
It can only be done as an option to PubSub which takes a map of topic name -> score and to get the topic name we need this exported.

Reply:
ok. I see. Thanks for the explanation. APPPPROOOOOVED